tests/model_registry/python_client/test_model_registry_creation.py

```diff
@@ -86,7 +87,42 @@ def test_model_registry_operator_env(
         if not namespace_env:
             pytest.fail("Missing environment variable REGISTRIES_NAMESPACE")
 
-    # TODO: Edit a registered model
-    # TODO: Add additional versions for a model
-    # TODO: List all available models
-    # TODO: List all versions of a model
+    def test_model_registry_grpc_container_removal(self, model_registry_deployment_containers: dict[str, Any]):
+        """
+        RHOAIENG-26239: Test to ensure removal of grpc container from model registry deployment
+        Steps:
+            Create metadata database
+            Deploys model registry using the same
+            Check model registry deployment for grpc container. It should not be present
+        """
+
+        for container in model_registry_deployment_containers:
+            if "grpc" in container["name"]:
+                pytest.fail(f"GRPC container found: {container}")
+
+    @pytest.mark.parametrize(
+        "endpoint",
+        [
+            pytest.param(
+                "oauth/healthz",
+            ),
+            pytest.param(
+                "readyz/isDirty",
+            ),
+        ],
+    )
+    def test_model_registry_endpoint_response(
+        self, model_registry_rest_url: str, model_registry_rest_headers: dict[str, str], endpoint: str
+    ):
+        """
+        RHOAIENG-26239: Test to ensure model registry endpoints are responsive
+        Steps:
+            Create metadata database
+            Deploys model registry using the same
+            Ensure endpoint is responsive via get call
+        """
+        # RHOAIENG-26239
```

Comment:
you have already in the docstring

Reply:
do you want to leave a TODO to add another parameter when https://issues.redhat.com/browse/RHOAIENG-29122 is completed?

Reply:
No need. We should have a test story associated for that one.